### PR TITLE
0.60.1 Release notes

### PR DESCRIPTION
**Pull request by MoOx**
@sgny @cknitt & others, does this looks good?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
 # Changelog of `reason-react-native`
 
+## 0.60.1 - 2019-10-25
+
+Probably the latest release for React Native 0.60. We will focus in 0.61.
+
+### 💥 Breaking changes (minor)
+
+Sorry, a mix & bugfix & changes! This should be easy to handle for you.
```
[the remark this answers is not in the document]
Comment (MoOx) [reply CHANGELOG.md]: #612 seems more like a simple fix to me?

Anyway, this sentence is for the 2 things below, which are a fix for the binding that can make your app stop compiling if you don't adjust the call to this specific things (if used)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,76 @@
 # Changelog of `reason-react-native`
 
+## 0.60.1 - 2019-10-25
+
+Probably the latest release for React Native 0.60. We will focus in 0.61.
```
[the remark this answers is not in the document]
Comment (MoOx) [reply CHANGELOG.md]: FYI, you can use 

image

To directly propose fix that can be merged via GitHub UI.

[the remark this answers is not in the document]
Comment (MoOx) [reply CHANGELOG.md]:
```suggestion
Probably the latest release for React Native 0.60. We will focus on 0.61.
```

image

[the remark this answers is not in the document]
Comment (MoOx) [reply CHANGELOG.md]: Then 
image

Tada!

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cknitt) [reply CHANGELOG.md]: Cool, didn't know that feature (not using Github at work). 👍

Yes, #621 is a fix, but code using the event listener `remove` functions that compiled and worked previously (as according to @sgny the `remove` functions do not really require the correct `this` binding) may need changes.

[the remark this answers is not in the document]
Comment (cknitt) [reply CHANGELOG.md]:
```suggestion
Sorry, a mix of bugfixes & changes! This should be easy to handle for you.
```

**Comment by MoOx** [comment]
@sgny any feedback before we cut this release?

**Pull request merged**